remotetests/remotetest.js

```diff
@@ -885,6 +888,38 @@ describe('Caches', function() { //  it
       }
     });
   });
+
+  it('Create an Cache Resource with description and expiry',function(done){
+    var opts = baseOpts();
+    opts.cache = CACHE_RESOURCE_WITH_EXPIRY_NAME;
+    opts.description = CACHE_RESOURCE_WITH_EXPIRY_DESCRIPTION;
+    opts.cacheExpiryInSecs = CACHE_RESOURCE_WITH_EXPIRY_TIMEOUT;
+    apigeetool.createcache(opts,function(err,result) {
+      if (verbose) {
+        console.log('Create Cache result = %j', result);
+      }
+      if (err) {
+        done(err);
+      } else {
+        done()
+      }
+    });
+  });
+
+  it('Delete Cache Resource having with description and expiry',function(done){
+    var opts = baseOpts();
+    opts.cache = CACHE_RESOURCE_WITH_EXPIRY_NAME;
+    apigeetool.deletecache(opts,function(err,result) {
+      if (verbose) {
+        console.log('Delete Cache result = %j', result);
+      }
+      if (err) {
+        done(err);
+      } else {
+        done()
+      }
+    });
+  });
```

Comment:
This shouldn't be a test because we're not really testing anything specific to your PR.  As mentioned above, I'd rather see tests that create/get/verify/delete _(in each test)_ instead of having a singular test to create cache(s) and a singular test to delete cache(s).

Reply:
Currently the lib only supports create cache and delete cache. We would need to add retrieve cache command to verify the cache configuration.
Is it ok to update test with create and delete alone for these 3 scenarios?

Reply:
Added singular test cases for below scenarios.

- Create cache with cacheExpiryByDate
- Create cache with cacheExpiryInSecs
- Create cache with both cacheExpiryByDate and cacheExpiryInSecs